sunpy/map/sources/soho.py

```diff
@@ -33,15 +34,31 @@ class EITMap(GenericMap):
     * `SOHO EIT User Guide <https://umbra.nascom.nasa.gov/eit/eit_guide/>`_
 
     """
+    def _handle_deprecated_ctypes(self, ctype1, ctype2):
+        """
+        Override the default implementation to handle EITMAP-specific logic for CTYPE values.
+        """
+        if ctype1.lower() in ("solar-x", "solar_x"):
+            ctype1 = 'HPLN-TAN'
+
+        if ctype2.lower() in ("solar-y", "solar_y"):
+            ctype2 = 'HPLT-TAN'
+
+        return ctype1, ctype2
 
     def __init__(self, data, header, **kwargs):
         super().__init__(data, header, **kwargs)
 
+        ctype1 = self.meta.get('ctype1', None)
+        ctype2 = self.meta.get('ctype2', None)
+
+        self._handle_deprecated_ctypes(ctype1, ctype2)
```

Comment:
The changes we want should not modify the init of any of the map sources.

We need to check if that approach works.

Reply:
yep got it